tools/plugins/com.liferay.ide.server.tomcat.ui/src/com/liferay/ide/server/tomcat/ui/editor/LiferayServerSettingsEditorSection.java

```diff
@@ -962,6 +962,19 @@ else if (path.equals(installDirPath))
 
 			}
 			*/
+            String memoryValue = memoryArgs.getText();
+            String[] memory = memoryValue.split( " " ); //$NON-NLS-1$
+            if( memory.length < 2 )
+            {
+                return new IStatus[] {
+                    new Status( IStatus.ERROR, LiferayServerUI.PLUGIN_ID, "error memeory format" ) }; //$NON-NLS-1$
+            }
+            if( !( ( memory[0].startsWith( "-Xmx" ) || memory[0].startsWith( "-XX:MaxPermSize" ) ) && //$NON-NLS-1$ //$NON-NLS-2$
```

Comment:
We need to support all possible memory arguments, so others than just -Xmx, so do some research for which ones should be supported.